trigger/context.go

```diff
@@ -25,3 +25,22 @@ func HandlerFromContext(ctx context.Context) (*HandlerInfo, bool) {
 	u, ok := ctx.Value(handlerKey).(*HandlerInfo)
 	return u, ok
 }
+
+// This allows trigger developer to pass request parameters to handler w/o going through output mapper, e.g.,
+//   ctx := trigger.NewContextWithValues(context.Background(), values)
+//   results, err := handler.Handle(ctx, triggerData.ToMap())
+
+type valueKey string
+
+const contextValueKey valueKey = "RequestParams"
```

Comment:
Should we rename `RequestParams` to `data` or `values`?

Reply:
renamed valueKey to "values"